### PR TITLE
feat(binar-car-rental): checkout page

### PR DESCRIPTION
**Pull request by ahmadsyahrizar**

### CODE REVIEW

File: checkout.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Checkout Page</title>
+</head>
+<body>
```
Comment (Justintnwj) [inline checkout.html]: tolong diwarnain biru

File: checkout.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+   <p class="lead">
+    <a class="btn btn-primary btn-lg" href="Jumbo action link" role="button">Jumbo action name</a>
+   </p>
+  </div>
+
+  <!-- card 1 -->
+  <div class="card">
+   <img class="card-img-top" src="holder.js/100x180/" alt="">
+   <div class="card-body">
+    <h4 class="card-title">Title</h4>
```
Comment (wahyupambudi) [inline checkout.html]: add class text-danger on h4

File: checkout.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+   <button class="navbar-toggler d-lg-none" type="button" data-toggle="collapse" data-target="#collapsibleNavId" aria-controls="collapsibleNavId"
+       aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+   </button>
+   <div class="collapse navbar-collapse" id="collapsibleNavId">
+    <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
+     <li class="nav-item active">
+      <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+     </li>
+     <li class="nav-item">
```
Comment (joviantomd) [inline checkout.html]: delete this section

**Review by joviantomd — CHANGES_REQUESTED** [review]

File: checkout.html (hunk 2 above)
Comment (IdoMunthe) [reply checkout.html]: Thank you for reminding me!

File: checkout.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  </div>
+
+  <!-- card 2 -->
+  <div class="card">
+   <img class="card-img-top" src="holder.js/100x180/" alt="">
+   <div class="card-body">
+    <h4 class="card-title">Title</h4>
+    <p class="card-text">Text</p>
+   </div>
+  </div>
```
Comment (Zulqs) [inline checkout.html:60]: Where the footer??

File: checkout.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+   <hr class="my-2">
+   <p>More info</p>
+   <p class="lead">
+    <a class="btn btn-primary btn-lg" href="Jumbo action link" role="button">Jumbo action name</a>
+   </p>
+  </div>
+
+  <!-- card 1 -->
+  <div class="card">
+   <img class="card-img-top" src="holder.js/100x180/" alt="">
```
Comment (hafidluqman50) [inline checkout.html:47]: remove attribute empty alt, it's dirty code

File: checkout.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+   <h1 class="display-3">Jumbo heading</h1>
+   <p class="lead">Jumbo helper text</p>
+   <hr class="my-2">
+   <p>More info</p>
+   <p class="lead">
+    <a class="btn btn-primary btn-lg" href="Jumbo action link" role="button">Jumbo action name</a>
+   </p>
+  </div>
+
+  <!-- card 1 -->
```
Comment (adhsen456) [inline checkout.html]: delete the comment. it's unnecessary

File: checkout.html [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+   <img class="card-img-top" src="holder.js/100x180/" alt="">
+   <div class="card-body">
+    <h4 class="card-title">Title</h4>
+    <p class="card-text">Text</p>
+   </div>
+  </div>
+
+  <!-- card 2 -->
+  <div class="card">
+   <img class="card-img-top" src="holder.js/100x180/" alt="">
```
Comment (Jeremiapane) [inline checkout.html]: Please change the dimensions of the card image

**Review by Jeremiapane — CHANGES_REQUESTED** [review]

File: checkout.html (hunk 3 above)
Comment (ahmadsyahrizar) [reply checkout.html]: done.

File: checkout.html (hunk 5 above)
Comment (ahmadsyahrizar) [reply checkout.html:47]: instead of remote alt attribute, I prefer to add value to it, so it will make SEO score to be better.

**Review by joviantomd — APPROVED** [review]

**Review by Zulqs — APPROVED** [review]
Great, thanks!

**Review by Justintnwj — APPROVED** [review]

**Review by Jeremiapane — APPROVED** [review]

**Review by hafidluqman50 — APPROVED** [review]
good

**Review by adhsen456 — APPROVED** [review]

**Review by IdoMunthe — APPROVED** [review]

**Review by wahyupambudi — APPROVED** [review]

**Pull request merged**